core-services/prow/02_config/openshift/cluster-kube-apiserver-operator/_pluginconfig.yaml

```diff
@@ -6,6 +6,22 @@ label:
       - wangke19
       - gangwgr
       label: cherry-pick-approved
+    - allowed_users:
+      - adambkaplan
```

Comment:
I should not be on this list - Build + Jenkins team doesn't touch the kube apiserver (or operator)